src/main/scala/org/allenai/scholar/eval/CreateGoldData.scala

```diff
@@ -65,34 +66,26 @@ object DblpFromPipeline extends App with CreateGoldData {
 
   import MetadataParser._
 
-  val rand = new Random()
-  var size = 1000.0
-  var count = 1.0
-  val clusters = new ListBuffer[Cluster]
   val keepId: String => Boolean = {
     val idFile = new File("src/main/resources/raw/dblp/ids.txt")
-    if (idFile.exists) {
-      // Use ID file if it exists
-      val ids = Resource.using(Source.fromFile(idFile)) { src => src.getLines.toSet }
-      s: String => ids(s)
-    } else {
-      // Resevoir sample 1000 papers
-      s: String => (rand.nextDouble < size / count)
-    }
+    val ids = Resource.using(Source.fromFile(idFile)) { src => src.getLines().toSet }
+    s: String => ids(s)
   }
```

Comment:
I removed the sampling stuff, because there are only 1000 papers anyways. Why bother with sampling?